porcupine/default_filetypes.toml

```diff
@@ -313,6 +344,20 @@ tabs2spaces = true
 filename_patterns = ["*.toml"]
 pygments_lexer = "pygments.lexers.TOMLLexer"
 comment_prefix = '#'
+[TOML.tree_sitter]
```

Comment:
This no longer works. I have switched to putting token mappings into a separate yml file.